docs/machine-learning/tutorials/image-classification.md

```diff
@@ -120,10 +120,10 @@ The training and testing images are located in the assets folders that you'll do
 
     * In Solution Explorer, right-click on your project and select **Manage NuGet Packages**.
     * Choose "nuget.org" as the Package source, select the Browse tab, search for **Microsoft.ML**.
-    * Click on the **Version** drop-down, select the **1.4.0** package in the list, and select the **Install** button.
+    * Select the **Install** button.
```

Comment:
Should we still mention which version (1.5.0)?

Reply:
Done